crates/consensus/src/pure/eip4844.rs

```diff
@@ -0,0 +1,150 @@
+use crate::constants::eip4844::{
```

Comment:
Nit I'd maybe group by EIP instead of by constants as it seems nicer to import everything EIP related in one call

Reply:
do you have a better name for this class of things, btw? the pure functions and constants?